src/PKSim.Core/Services/RenameAbsolutePathVisitor.cs

```diff
@@ -45,7 +45,9 @@ private bool isAbsolutePath(string pathAsString)
 
       private void renameAbsolutePathIn(IFormula formula)
       {
-         if (formula == null || !formula.IsExplicit()) return;
```

Comment:
this was only checking path for explicit formula. It should actually do it for all formula except constants